### PR TITLE
Paul Alves homework submission

### PR DESCRIPTION
**Pull request by gummywormz**
Working on the assignment was easy as I've had plenty of prior experience in both Github and Markdown. There are some inconsistencies between Github's and RStudio's rendering of markdown files however such as alternate image text and header spacing.

I started by cloning the repository to my account on the github website. Then I loaded Github for Windows to clone the repository to my hard drive. Then I opened the folder as a project in RStudio. I edited the file in that program. Finally, I committed and synched changes to the folder using Github for Windows and then opening this pull request via the github website.

### CODE REVIEW

**Comment by Bobicheng-Zhang** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |           |       X     |            |
| **Ease of access**          |     X      |              |            |

Remarks:
* All the codes are really clean and simplistic, and everything is very much functioning and easy to access
* I am sure you could have done a little more to make it look cooler and more creative given your prior experience

**Comment by rachelyhkim** [comment]
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |           |      x        |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* I'm glad you were able to catch the spacing issue with the headers in your second commit—this was also something that I had to repeatedly fix in my own work.

* As the above peer reviewer noted, I also appreciated the simplicity of your code—a skill that I think must serve you well as an experienced coder. I also would have liked to see a more complex or creative biography given your experience.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+I am a political science student with a strong tech background. I have been programming since I was 10 years old and been working with computers since even earlier. I enjoy fixing computers and doing video game design. 
+
+## Current Skills
+* ArcGIS
+* Stata
+* Microsoft Excel
+* Java
+* Haxe
+* Python
+* Android _(when it doesn't take 3 days to install the development kit)_
```
Comment (mayank-kgp) [inline README.md:14]: It took 4 days for me.

**Comment by mayank-kgp** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |           |      x        |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* You don't need to commit all files such as .gitignore or .Rproj files. Just commit those files which are necessary.
* Page is simple and easy to read. Coding is good without any error.

**Pull request closed without merging**